### PR TITLE
Pipeline improvements

### PR DESCRIPTION
**Pull request by anitacaron**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -17,21 +17,15 @@ kidney-annotations.owl: kidney-ont.owl kidney-seed.txt
...
+	$(ROBOT) merge -i uberon-base.owl -i cl-base.owl -o $@
 
-materialize.ttl: merged_imports.owl
-	$(ROBOT) materialize --reasoner ELK --input $< --term BFO:0000050 --output $@
-
-.PHONY: materialize.ttl
-
-materialize-direct.nt: materialize.ttl
+materialize-direct.nt: merged_imports.owl
 	$(RG) --ontology-file $< --output-subclasses true --output-file $@
```
Comment (balhoff) [inline Makefile:29]: If you want here you can do `$(RG) --property 'http://purl.obolibrary.org/obo/BFO_0000050' etc.`

Comment (anitacaron) [reply Makefile]: I added but it didn't change the graph result.

**Comment by dosumis** [comment]
Pasting in Jims' review notes for reference:

> > The materialize step is not needed for relation-graph (it does its own thing like this internally), but maybe it’s needed for other steps? https://github.com/anitacaron/souffle_test/blob/cc8b7440af21d7e6f0aa1223b58c4706e65c17db/Makefile#L29-L35

> >  I think you will end up with all possible (i.e. redundant) part_of relations (depending on input as discussed above). Am I missing where it would get pruned back? I’m assuming you would want it pruned to non-redundant. Here’s how I would see the workflow:
> > * Input complete ontology to relation-graph, with CLI arg requesting only relation part_of
> > * Feed this output into your souffle script
> > * Include code in the souffle script which removes redundant part_of relations: https://github.com/INCATools/ubergraph/blob/a27eda415f2379ffdcdc563feaa466c0e3a16830/prune.dl#L29-L35
> > prune.dl

> **Anita**: Doesn't the reduce do the job to non redundant here? https://github.com/anitacaron/souffle_test/blob/cc8b7440af21d7e6f0aa1223b58c4706e65c17db/Makefile#L63
> **Jim:** Ah, I think you’re right  
> I don’t know exactly how the outputs compare

**Comment by dosumis** [comment]
### How the two graphs compare:

I think the new code is either under inferring or over stripping.

Examples:



'macula densa epithelial cell' is a singleton in the new graph, but in the new graph is linked to nephron tubule in the old graph

image
(BTW - more PDFs are better than pngs for reports like this as they are text searchable)

This looks like a simple failure of gap filling as 'macula densa' is not in the seed:

image

Another, more obvious example, the new graph has no cell types directly under kidney, but the old graph does:

image

In this case a direct relationship seems to have been lost: 
'renal interstitial pericyte' has:
image

**Comment by dosumis** [comment]
@balhoff - any idea what's going on?  I think I'm more comfortable relying on a materialization with robot + ELK.

**Pull request merged**